test-alerts.cpp

```diff
@@ -6,3 +6,74 @@
 TEST_CASE("infers the breach according to limits") {
   REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
 }
+
+TEST_CASE("Check temperature classification") {
+  std::unordered_map <int ,std::pair<int, int>> testTempClassify;
+  classifyTemperatureBreach(testTempClassify);
```

Comment:
Test cases are acceptable for the method implemented. Naming of the method should be improved.